subsys/bluetooth/controller/ll_sw/ull_iso.c

```diff
@@ -196,41 +186,6 @@ uint8_t ll_read_iso_tx_sync(uint16_t handle, uint16_t *seq,
 	return BT_HCI_ERR_UNKNOWN_CONN_ID;
 }
 
-/* Must be implemented by vendor */
-__weak bool ll_data_path_sink_create(uint16_t handle,
-				     struct ll_iso_datapath *datapath,
-				     isoal_sink_sdu_alloc_cb *sdu_alloc,
-				     isoal_sink_sdu_emit_cb *sdu_emit,
-				     isoal_sink_sdu_write_cb *sdu_write)
-{
-	ARG_UNUSED(handle);
-	ARG_UNUSED(datapath);
-
-	*sdu_alloc = NULL;
-	*sdu_emit  = NULL;
-	*sdu_write = NULL;
-
-	return false;
-}
-
-/* Could be implemented by vendor */
-__weak bool ll_data_path_source_create(uint16_t handle,
```

Comment:
this function was moved to the vendor specific file, why this one and not the others?